packages/cozy-client/src/models/dacc.js

```diff
@@ -131,7 +131,7 @@ export const fetchAggregatesFromDACC = async (
 ) => {
   try {
     const aggregateParams = buildAggregateParams(params)
-    await client
+    return client
```

Comment:
```suggestion
    return await client
```
Sinon le catch ici ne servira plus

Reply:
Ah bon ? Je veux bien une explication du pourquoi :)

Reply:
Très clair, merci. Il me semblait qu'on m'avait dit qu'on pouvait virer le await sur le `return`, mais c'était peut être un cas différent

Reply:
En effet, on pourra en discuter à l'occasion ;)